tkp/accessors/fitsimage.py

```diff
@@ -124,23 +124,34 @@ def parse_frequency(self):
         try:
             header = self.header
             if header['TELESCOP'] in ('LOFAR', 'AARTFAAC'):
-                freq_eff = header['RESTFRQ']
-                if 'RESTBW' in header:
-                    freq_bw = header['RESTBW']
+                if 'RESTFRQ' in header:
+                    freq_eff = header['RESTFRQ']
+                    if 'RESTBW' in header:
+                        freq_bw = header['RESTBW']
```

Comment:
You could probably tidy this up a bit if you drop this telescope line and just check for `RESTFRQ` and `RESTBW` in the header (maybe the LOFAR/AARTFAAC have been standardised such that you no longer need these?)

You could then avoid the duplication of the code below (the CTYPE3 and CTYPE4 checks).

Essentially: 
```
if `RESTFRQ` in header:

elif <CTYPE3 check>

elif <CTYPE4 check>

else <RESTFREQ fallback>
```

Reply:
Thanks for checking the code so quickly! 
That looks like an excellent idea. LOFAR and AARTFAAC12 are typically using WSClean now so should be standard, but lets keep RESTFRQ and RESTFREQ for old images. I'll make those changes today